### DISCUSSION TITLE
Exec in Integrated Terminal

### DISCUSSION THREAD

**Discussion by stevelaclasse** [getseabird/seabird#65] [General]
Hi everyone and thanks for this awesome project.

I saw on this page : [https://getseabird.github.io/](https://getseabird.github.io/), that we could exec into a container. Here is a picture showing that : ![Alt text](https://getseabird.github.io/images/screenshots/0.2-term.png "Integrated Terminal") .

I have download the latest release : **v0.2.2** and could not find that option. 

Is the feature to Exec into a running container already available or it is still in development ?

Thanks

**Comment by jgillich** [comment]
The terminal is currently not supported on Windows. This was mentioned in the v0.1 announcement, but it probably should be documented elsewhere too. This isn't a high priority for me right now, but contributions are welcome. I made #66 if anyone's interested.
